packages/eslint-plugin/src/rules/no-hardcoded-color.test.ts

```diff
@@ -0,0 +1,40 @@
+import { tester } from 'src/test-utils'
+import rule, { RULE_NAME } from './no-hardcoded-color'
+
+const imports = `import { css } from './panda/css'
+import { Circle } from './panda/jsx'
+`
+
+// Watch out for color opacity in the future
+const valids = [
+  'const styles = css({ color: "red.100" })',
+  '<div className={css({ background: "green.300" })} />',
+  '<Circle _hover={{ borderColor: "gray.100" }} />',
+  '<Circle _hover={{ bg: "gray.100" }} />',
+  '<Circle _hover={{ bgColor: "gray.100" }} />',
+]
+
+const invalids = [
+  'const styles = css({ color: "skyblue" })',
+  '<div className={css({ background: "#111" })} />',
+  '<Circle _hover={{ borderColor: "rgb(1, 1, 1)" }} />',
+  '<Circle _hover={{ bg: "hsl(0deg, 0%, 7%)" }} />',
+  '<Circle _hover={{ bgColor: "rgba(17, 17, 17, 1)" }} />',
+]
```

Comment:
we should allow using the `[xxx]` escape-hatch syntax

Reply:
Might need to discuss this more. But IMO, that's exactly something we want eslint to prevent us from doing. 
cc @segunadebayo